### PR TITLE
Corrected useTokenBalance() arguments order

### PR DESCRIPTION
**Pull request by antooni**
In Core API hook is documented correctly, only there was an error.

### CODE REVIEW

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -137,11 +137,11 @@ Provides a way to fetch balance of ERC20 token specified by ``tokenAddress`` for
 
   export function TokenBalance() {
     const { account } = useEthers()
-    const tokenBalance = useTokenBalance(account, DAI)
+    const tokenBalance = useTokenBalance(DAI, account)
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:140]: 👍

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -137,11 +137,11 @@ Provides a way to fetch balance of ERC20 token specified by ``tokenAddress`` for
...
 
     return (
       <div>
-        {tokenBalance && <p>Balance: {formatUnits(tokenBalance, 18)}</p>}
+        {tokenBalance && <p>Balance: {formatUnits(tokenBalance as any, 18)}</p>}
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:144]: Why as any?

Comment (antooni) [reply docs/source/getting-started.rst:144]: Without it I got an error:
  " Argument of type 'BigNumber' is not assignable to parameter of type 'BigNumberish' "

Comment (noobshow) [reply docs/source/getting-started.rst:144]: I don't get such error here, do you have another BigNumber library imported in the scope? Maybe other BigNumber library

Comment (marekkirejczyk) [reply docs/source/getting-started.rst:144]: You should be able to fix it by updating to the newest ethers and useDApp.

Comment (noobshow) [reply docs/source/getting-started.rst:144]: Sometimes, I get such error... But, I try to check properly and most times saw I'm not doing something right

Comment (marekkirejczyk) [reply docs/source/getting-started.rst:144]: There was in the older version problem when app was using ethers 5.1.* and useDApp was using ethers 5.0.*
Latest useDApp version is using 5.1.* and so should your app use. With that it should all work and there should be no type mismatches.

**Review by marekkirejczyk — CHANGES_REQUESTED** [review]
Remove `as any`

**Review by marekkirejczyk — APPROVED** [review]

**Review by marekkirejczyk — APPROVED** [review]

**Pull request merged**